### PR TITLE
Add GitHub workflows to FE hamburger

### PR DESCRIPTION
**Pull request by ioedeveloper**
#### Description
Adds 2 new workflows (ts-sol-test and slither) to FE hamburger UI.

Closes https://github.com/ethereum/remix-project/issues/3182

#### Discuss
- Find a better UI for listing GHactions in hamburger. For now it uses the default. @LianaHus @ryestew @joeizang @vermouth22 
I tried using a nested/multi-level dropdown for items, but the UI was hidden because of side-panel iframe overlap with the main panel iframe.
- Should GHaction items be listed only if the project is a git project?

#### TODO
- [x] Fix issue with remix-url-resolver so remix reward tests can pass.
- [x] Publish v1 of EthereumRemix/ts-sol-test to marketplace and update readme.

### CODE REVIEW

**Review by Aniket-Engg — COMMENTED** [review]
Looks fine to me. Can you add steps to test if all these actions actually work on Github?

**Comment by ioedeveloper** [comment]
> Looks fine to me. Can you add steps to test if all these actions actually work on Github?

- Clone a git repo
- Add github workflows by selecting workflow option from FE hamburger
- Push to github using dGit plugin

File: libs/remix-ui/workspace/src/lib/utils/constants.ts [hunk 1]
```diff
@@ -1 +1,49 @@
...
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Run SUT Action
+      uses: EthereumRemix/sol-test@v1
+      with:
+        test-path: 'tests'
+        compiler-version: '0.8.15'
+`
+export const tsSolTestYml = `
+name: ts-sol-test
```
Comment (Aniket-Engg) [inline libs/remix-ui/workspace/src/lib/utils/constants.ts:20]: Considering other jobs, name should be update to be more meaningful

File: libs/remix-ui/workspace/src/lib/utils/constants.ts [hunk 2]
```diff
@@ -1 +1,49 @@
...
+on: [push]
+
+jobs:
+  run_sample_test_job:
+    runs-on: ubuntu-latest
+    name: A job to run mocha and chai tests for solidity on github actions CI
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Run SUT Action
```
Comment (Aniket-Engg) [inline libs/remix-ui/workspace/src/lib/utils/constants.ts:30]: SUT refers to Solidity Unit Testing job, it should be updated

**Review by Aniket-Engg — COMMENTED** [review]
To push the workflows, Github token on Remix should have `workflow` scope. It should be mentioned somewhere

**Comment by ioedeveloper** [comment]
> To push the workflows, Github token on Remix should have `workflow` scope. It should be mentioned somewhere

Where do you want it to be mentioned @Aniket-Engg ?

**Review by yann300 — APPROVED** [review]

**Pull request merged**